chainer/functions/pooling/upsampling_2d.py

```diff
@@ -103,7 +102,24 @@ def forward_gpu(self, x):
                                self.outh, self.outw)
         return up_y,
 
-    def backward_cpu(self, x, gy):
+    def backward(self, indexes, grad_outputs):
+        return Upsampling2DGrad(self).apply(grad_outputs)
+
+
+class Upsampling2DGrad(function_node.FunctionNode):
```

Comment:
Is it correct that this function has the gradient being always zero?

Reply:
Sorry, I should have implemented the backward for this class. So, I added it.